src/Resources/config/services.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <parameters>
+        <parameter key="app.taxation.eu_zone_code">TAX_EU</parameter>
```

Comment:
It would be better to provide no default value for this parameter. Doing so the developer is forced to define it and to think about it. But probably you defined it to avoid BC break, am I right?

Anyway I'd document this in the README. We should explain the new feature about tax calculation saying that Sylius should configured with a EU zone with this code.